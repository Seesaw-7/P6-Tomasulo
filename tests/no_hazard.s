addi x2, x2, 0x1 #
nop
nop
nop
nop
add x3, x3, x2 #
addi x4, x4, 0x8 #
addi x5, x5, 0x8 #
addi x10, x10, 0x1 #
slti x11, x10, 16 #
addi x6, x6, 0x8 #
wfi
