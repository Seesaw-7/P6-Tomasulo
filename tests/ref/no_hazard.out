Loading memory file: output/no_hazard.hex
Using writeback output file: output/no_hazard.wb
Using pipeline output file: output/no_hazard.ppln
Using pipeline output file: output/no_hazard.reg
@@
@@
@@                     0  Asserting System reset......
@@
@@                   100 : System STILL at reset, can't show anything
@@
@@
@@                   200 : System STILL at reset, can't show anything
@@
@@
@@                   300 : System STILL at reset, can't show anything
@@
@@                   360  Deasserting System reset......
@@
@@
@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 0000001300110113 : 81605493011
@@@ mem[    8] = 0000001300000013 : 81604378643
@@@ mem[   16] = 002181b300000013 : 9431280030580755
@@@ mem[   24] = 0082829300820213 : 36735314852708883
@@@ mem[   32] = 0105259300150513 : 73506282213278995
@@@ mem[   40] = 1050007300830313 : 1175439996673524499
@@@
@@@
@@                  6320 : System halted
@@
@@@ System halted on WFI instruction
@@@
@@
@@  60 cycles / 11 instrs = 5.454545 CPI
@@
@@  590.00 ns total time to execute
@@

- testbench/testbench.sv:313: Verilog $finish
- S i m u l a t i o n   R e p o r t: Verilator 5.026 2024-06-15
- Verilator: $finish at 735ns; walltime 0.001 s; speed 0.000 s/s
- Verilator: cpu 0.000 s on 1 threads; alloced 121 MB
